spec/models/king_spec.rb

```diff
@@ -4,4 +4,20 @@
   it 'has type of King' do
     expect(subject.type).to eq 'King'
   end
+
+  describe '#valid_move?' do
+    it 'is a public instance method' do
+      expect(subject).to respond_to :valid_move?
```

Comment:
I think this test is unneeded.

A bit redundant since it gets called in the other tests (and would fail there if not defined). :)